### ISSUE TITLE
Camera statement is not properly ran in rollback

### ISSUE THREAD

**Issue by brimbel** [renpy/renpy#5710]
Tested on 8.3.0.24081503+nightly

```
label start:

    scene bg room
    show eileen happy

    "START: 1"

    camera:
        xpos 140 ypos -150
    with dissolve

    camera:
        ease 1.25 yoffset 240
    pause 1.25

    "2"

    "3"

    camera

    "RESET: 4"

    show eileen:
        anchor (0.5,0.0) xpos 140 ypos -150
    with dissolve

    show eileen:
        ease 1.25 yoffset 240
    pause 1.25

    "5"

    "6"

    return
```

https://github.com/user-attachments/assets/e1dd8303-72f8-4a9e-895e-4fee6b0e2a2e

When rolling back from 3 to 2, `camera: xpos 140 ypos -150` is not ran, but when rolling back from 6 to 5, `show eileen: anchor (0.5,0.0) xpos 140 ypos -150` is ran. Ideally, the camera statement should work like the show statement, or else every camera statement would need to state all of its properties lest it break in rollback.

**Comment by renpytom** [comment]
Fixed in a86cc5582bb3af715b772293ab76f597f23c31c4

**Comment by brimbel** [comment]
Tested on 8.3.0

When swapping out the `with dissolve` statements for `pause` statements, the same issue occurs

```
label start:

    scene bg room
    show eileen happy

    "START: 1"

    camera:
        xpos 140 ypos -150
    pause 1.0

    camera:
        ease 1.25 yoffset 240
    pause 1.25

    "2"

    "3"

    camera

    "RESET: 4"

    show eileen:
        anchor (0.5,0.0) xpos 140 ypos -150
    pause 1.0

    show eileen:
        ease 1.25 yoffset 240
    pause 1.25

    "5"

    "6"

    return
```

https://github.com/user-attachments/assets/cb45f998-01ad-457d-bd03-f3830930c4c1
